ddtrace/tracer/span_test.go

```diff
@@ -115,6 +115,19 @@ func TestSpanFinishWithError(t *testing.T) {
 	assert.NotEmpty(span.Meta[ext.ErrorStack])
 }
 
+func TestSpanFinishWithErrorNoBacktrace(t *testing.T) {
```

Comment:
Would it be worthwhile to have two subtests here? One checking that the stack trace is there and the other one that it’s not? (Like your current test does). If there already is a test that asserts that then nevermind, just ignore this comment.

Reply:
There's a test just above.